airbyte-cdk/java/airbyte-cdk/gcs-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/gcs/parquet/GcsParquetWriter.kt

```diff
@@ -46,7 +46,7 @@ class GcsParquetWriter(
 
     init {
         val outputFilename: String =
-            BaseGcsWriter.Companion.getOutputFilename(uploadTimestamp, S3Format.PARQUET)
+            BaseGcsWriter.Companion.getOutputFilename(uploadTimestamp, FileUploadFormat.PARQUET)
         outputPath = java.lang.String.join("/", outputPrefix, outputFilename)
         LOGGER.info(
             "Storage path for stream '{}': {}/{}",
```

Comment:
This whole class and the one above look suspicious. This init seems to be doing the same thing except for odd differences, like using an s3:// address here.... I'm definitely in need of understanding those thing better...

Reply:
(just to be clear, I understand that this came from the autoformatting, so I'm not expecting any changes from you)